main.tf

```diff
@@ -80,3 +80,37 @@ resource "google_cloudbuildv2_connection" "cloudbuild_github_project_connection"
   }
   depends_on = [google_secret_manager_secret_iam_policy.policy]
 }
+
+resource "google_cloudbuild_trigger" "filename-trigger" {
+  location = var.gcp_region
```

Comment:
Not sure if we want to change regions due to this:
https://cloud.google.com/build/docs/locations#restricted_regions_for_some_projects
Restricted regions for some projects

> Depending on usage, certain projects may be restricted to only use Cloud Build in the following regions:
> 
> us-central1
> us-west2
> europe-west1
> asia-east1
> australia-southeast1
> southamerica-east1